src/System.Runtime.Numerics/tests/BigInteger/IsEven.cs

```diff
@@ -13,6 +13,7 @@ public class IsEvenTest
         private static int s_seed = 0;
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```

Comment:
I think we actually need to figure out why the behavior is different here. This one doesn't seem to be doing anything with internal state.